cluster/ci/monitoring/build_cop.json

```diff
@@ -685,7 +685,7 @@
          "thresholds": [ ],
          "timeFrom": null,
          "timeShift": null,
-         "title": "AWS Quota Leases by State",
+         "title": "Quota Leases by State: ${iaas}",
```

Comment:
It's currently impossible to get the human-readable values to display here like how they display for the picker. Oh well